bower.json

```diff
@@ -6,6 +6,12 @@
   "license": "GPLv3",
   "dependencies": {
     "jquery": "^3.1.1",
-    "bootstrap": "4.0.0-alpha.6"
+    "bootstrap": "4.0.0-alpha.6",
+    "socket.io-client": "1.7.2"
+  },
+  "overrides": {
+    "socket.io-client": {
+      "main": ["socket.io.js"]
+    }
```

Comment:
Can you specify why the overrides section is required with a comment?

I know json doesn't allow comments but there's a workaround. Insert a "_comment": "the overrides does X", right above it.